hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md

```diff
@@ -640,11 +640,11 @@ support other pre-configured queues to co-exist along with auto-created queues.
 
 * Configuring **legacy** `Auto-Created Leaf Queues` with `CapacityScheduler`
 
-The parent queue which has been enabled for auto leaf queue creation,supports
- the configuration of template parameters for automatic configuration of the auto-created leaf queues. The auto-created queues support all of the
- leaf queue configuration parameters except for **Queue ACL**, **Absolute
- Resource** configurations. Queue ACLs are
- currently inherited from the parent queue i.e they are not configurable on the leaf queue template
+The parent queue which has been enabled for auto leaf queue creation, supports
+ the configuration of template parameters for automatic configuration of the auto-created leaf queues. The auto-created queues support all the
+ leaf queue configuration parameters. There is one caveat to the QueueACLs: at auto queue creation the Queue ACLs in the leaf queue templates are not in effect yet,
```

Comment:
at auto queue creation -> I would change this to "at the time of queue auto creation" so that it refers more to the point (in time) when the queue is being created and not the topic of auto queue creation.